roaster-testsuite/roaster-testsuite-redis/src/main/java/hu/icellmobilsoft/roaster/testsuite/redis/RedisContainer.java

```diff
@@ -0,0 +1,42 @@
+/*-
+ * #%L
+ * Coffee
+ * %%
+ * Copyright (C) 2020 - 2022 i-Cell Mobilsoft Zrt.
+ * %%
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * #L%
+ */
+package hu.icellmobilsoft.roaster.testsuite.redis;
+
+import redis.clients.jedis.Jedis;
+
+/**
+ * Redis backend cache for tests
+ * 
+ * @author imre.scheffer
+ * @since 0.8.0
+ *
+ */
+public class RedisContainer {
```

Comment:
Ez az osztály biztos kell?

Reply:
Én úgy láttam, hogy a Jedis-t is lehetne static field-be rakni a teszt osztályban, és akkor nem kell ez az osztály. De így is jó.